ksqldb-engine/src/main/java/io/confluent/ksql/services/KafkaConsumerGroupClientImpl.java

```diff
@@ -73,4 +80,18 @@ public ConsumerGroupSummary describeConsumerGroup(final String group) {
       throw new KafkaResponseGetFailedException("Failed to describe Kafka consumer groups", e);
     }
   }
+
+  @Override
+  public Map<TopicPartition, OffsetAndMetadata> listConsumerGroupOffsets(final String group) {
+    try {
+      return ExecutorUtil.executeWithRetries(
+          () -> adminClient.get()
+              .listConsumerGroupOffsets(group)
+              .partitionsToOffsetAndMetadata()
+              .get(),
+          RetryBehaviour.ON_RETRYABLE);
+    } catch (final Exception e) {
+      throw new KafkaResponseGetFailedException("Failed to retrieve Kafka consumer groups", e);
+    }
```

Comment:
Can you extend this error handling, as suggestted above for another function, to return more helpful error messages to the user please?

 - add catch block for auth errors
 - add catch block to unwrap execution errors.